quil-rs/src/instruction/gate.rs

```diff
@@ -220,13 +226,13 @@ impl fmt::Display for GateSpecification {
                 writeln!(f)?;
             }
             GateSpecification::PauliSum(pauli_sum) => {
-                for term in &pauli_sum.0 {
+                for term in &pauli_sum.terms {
                     write!(f, "\t")?;
-                    for word in term.word.iter() {
+                    for (word, _) in term.arguments.iter() {
                         write!(f, "{word}")?;
                     }
                     write!(f, "{}", term.expression)?;
-                    for argument in term.arguments.iter() {
+                    for (_, argument) in term.arguments.iter() {
```

Comment:
Would it be useful to add methods `words` and `arguments` that return iterators for each of these? I know my suggest for the above conversation would be (slightly) improved with an `arguments()` method.

Reply:
Yeah, I like that idea, done.